frontend/src/bundles/landing/pages/landing.tsx

```diff
@@ -0,0 +1,428 @@
+                                        name={testimonial.name}
+                                        occupation={testimonial.head}
+                                        avatarImage={testimonial.avatar}
+                                        avatarBgColor={testimonial.bgColor}
+                                    />
+                                ),
+                            )}
+                        </div>
+                    </Fade>
+                    <Fade x={-200}>
+                        <div className="hidden flex-nowrap justify-center gap-4 md:flex">
+                            {TESTIMONIALS.slice(7, 10).map(
+                                (testimonial, index) => (
+                                    <TestimonialCard
+                                        key={index}
+                                        rating={5}
+                                        text={testimonial.text}
+                                        name={testimonial.name}
+                                        occupation={testimonial.head}
+                                        avatarImage={testimonial.avatar}
+                                        avatarBgColor={testimonial.bgColor}
+                                    />
+                                ),
+                            )}
+                        </div>
+                    </Fade>
+                </div>
+            </section>
+            <section className={getValidClassNames(styles.section)}>
+                <div className="bg-secondary flex flex-1 flex-col items-center justify-center p-4">
+                    <div className="max-w-[37.5rem] text-center leading-4">
+                        <h1 className="font-heavybold mb-[1rem] text-center text-[2.5rem] leading-[4rem] md:text-[3.5rem]">
+                            Ready to Ignite Your Fitness Journey?
+                        </h1>
+                        <p className="mb-[2.25rem] font-bold">
+                            Don’t just take our word for it, experience the
+                            transformation yourself! Download FitLife now and
+                            step into a world of personalized fitness and
+                            health. Your journey to a healthier, fitter you
+                            starts here.
+                        </p>
+                    </div>
+                    <Button
+                        size={ComponentSize.MEDIUM}
+                        label="Download Now and Get Fit"
+                        variant={ButtonVariant.PRIMARY}
+                        className="max-w-[15.5rem]"
+                        onClick={signUpHandler}
+                    />
+                </div>
+                <footer className="container mx-auto flex h-[40rem] flex-col-reverse items-center justify-evenly sm:h-[20rem] md:flex-row md:items-start md:pt-60">
+                    <div>
+                        <img src={logo} alt="Logo" />
```

Comment:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/72942456/cad60997-ba89-40f7-b276-094997d6ff70)
The logo in footer dissapears on light theme